### PR TITLE
config(kubeflow): kubeflow/pipelines require self approval for PRs

### PR DESCRIPTION
**Pull request by Bobgy**
According to https://github.com/kubernetes/test-infra/issues/19896#issuecomment-724413115,

We want to restore the behavior of always requiring self approval.

/assign @chaodaiG @Ark-kun

### CODE REVIEW

**Comment by Bobgy** [comment]
/hold
to wait for some internal discussion first

**Comment by Ark-kun** [comment]
+1 This is pretty useful. The authors often want to control when the PR is submitted.

**Review by chaodaiG — APPROVED** [review]

**Comment by Bobgy** [comment]
/unhold

Most people agree to try requiring explicit approval. Let's proceed and get more feedback over time.

**Pull request merged**
